core/indigo-core/molecule/src/smiles_saver.cpp

```diff
@@ -1992,6 +2042,9 @@ void SmilesSaver::_checkSRU()
 
     int i, j, k;
 
+    if (write_extra_info) // SRU will be handled better inside the extended block
+        return;
```

Comment:
please disable code below. what does "write_extra_info" mean. Is it extended block?